wren-ui/src/apollo/server/resolvers/projectResolver.ts

```diff
@@ -538,6 +640,7 @@ export class ProjectResolver {
       ctx,
       projectId: project.id,
     });
+    await schemaDetector.detectSchemaChange();
```

Comment:
After discussing with @fredalai , we'll remove this line for a better user experience.